### ISSUE TITLE
Your website

### ISSUE THREAD

**Issue by AlexNodex** [nestjs/nest#1590]
When visiting https://www.nestjs.com/ there is an SSL error.

You should either buy a valid cert for the www domain or fix your config to redirect to non www.

Didn't know where else to submit the issue

You're welcome :-)

**Comment by leszekkadlof** [comment]
Certificate seems fine. 

Can you check whether your system clock is set correctly?

**Comment by AlexNodex** [comment]
Every other website I visit in the world with https:// works fine.

![screen shot 2019-02-26 at 07 25 19](https://user-images.githubusercontent.com/17162626/53394635-be79a980-3997-11e9-8112-bfaae7e05c47.png)

**Comment by arkraft** [comment]
Checked it in Firefox and Chrome, in both cases the certificate is valid

**Comment by AlexNodex** [comment]
Chrome does a redirect back to the NON www tld which is why it works fine.

Chrome on iOS is not fine - see below screenshot.

Please note that the NON www domain IS FINE, anyone testing should make sure they're not being redirected.

Also see second screenshot from a `curl` command...

![img_1868](https://user-images.githubusercontent.com/17162626/53396221-d6ebc300-399b-11e9-9510-0773d25e2660.PNG)

![curl](https://user-images.githubusercontent.com/17162626/53396444-59748280-399c-11e9-8be4-368a0007d9b8.jpg)

**Comment by arkraft** [comment]
Yes, you are right. I used the NON www domain. The one with www does not work with Firefox and on Chroome i am redirected
